src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,11 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        return List.of(new Truck());

```

Comment:
The `get` method should return a list with 2-3 `Truck` instances, as suggested by the `MachineProducer` interface documentation. Consider adding more `Truck` instances to the list.